### PR TITLE
Add duration and time to the Kotlin serialization model

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This doesn't handle creating an appropriate question state or anything. Wasn't sure how best to handle this since it's super-weird but figured I would go ahead and add the serialization so that it was consistent. (I'll need to add it to the JsonModel-Swift project on iOS before adding the Swift input item.)

Also, we may want to consider adding a new question type with multiple input items rather than an input item that maps to a simple question? Idano. what do you think?

https://github.com/Sage-Bionetworks/JsonModel-Swift/pull/17

### CODE REVIEW

File: assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt [hunk 1]
```diff
@@ -12,25 +12,20 @@ import org.sagebionetworks.assessmentmodel.survey.AnswerType
 val inputItemSerializersModule = SerializersModule {
     polymorphic(InputItem::class) {
         subclass(CheckboxInputItemObject::class)
-        subclass(DateInputItemObject::class)
```
Comment (nategbrown9) [inline assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt:15]: Shouldn't TimeInputItemObject be here?

File: assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt [hunk 2]
```diff
@@ -12,25 +12,20 @@ import org.sagebionetworks.assessmentmodel.survey.AnswerType
...
         subclass(DoubleTextInputItemObject::class)
+        subclass(DurationInputItemObject::class)
         subclass(IntegerTextInputItemObject::class)
         subclass(StringTextInputItemObject::class)
         subclass(TimeInputItemObject::class)
         subclass(YearTextInputItemObject::class)
     }
     polymorphic(KeyboardTextInputItem::class) {
-        subclass(DateInputItemObject::class)
```
Comment (nategbrown9) [inline assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt:23]: Shouldn't TimeInputItemObject be here as well?

File: assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt (hunk 1 above)
Comment (syoung-smallwisdom) [reply assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt:15]: It was already in the model. This list is alphabetized.

File: assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt [hunk 3]
```diff
@@ -12,25 +12,20 @@ import org.sagebionetworks.assessmentmodel.survey.AnswerType
 val inputItemSerializersModule = SerializersModule {
     polymorphic(InputItem::class) {
         subclass(CheckboxInputItemObject::class)
-        subclass(DateInputItemObject::class)
         subclass(DoubleTextInputItemObject::class)
+        subclass(DurationInputItemObject::class)
         subclass(IntegerTextInputItemObject::class)
         subclass(StringTextInputItemObject::class)
         subclass(TimeInputItemObject::class)
```
Comment (syoung-smallwisdom) [inline assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt:19]: Was already in the list down here.

**Review by nategbrown9 — APPROVED** [review]
Looks good.

**Pull request merged**